Mathlib/MeasureTheory/Function/LpSeminorm.lean

```diff
@@ -921,6 +921,24 @@ theorem memℒp_map_measure_iff (hg : AEStronglyMeasurable g (Measure.map f μ))
   simp [Memℒp, snorm_map_measure hg hf, hg.comp_aemeasurable hf, hg]
 #align measure_theory.mem_ℒp_map_measure_iff MeasureTheory.memℒp_map_measure_iff
 
+theorem Memℒp.comp_of_map (hg : Memℒp g p (Measure.map f μ)) (hf : AEMeasurable f μ) :
+    Memℒp (g ∘ f) p μ :=
+  (memℒp_map_measure_iff hg.aestronglyMeasurable hf).1 hg
+
+theorem snorm_comp_measurePreserving {ν : MeasureTheory.Measure β} (hg : AEStronglyMeasurable g ν)
```

Comment:
Do we still need the overload `MeasureTheory.Measure` now that `Measure` has been removed from core? (Might require bumping to latest master).

Reply:
UPD: I merged the latest `master` but `_root_.Measure` is still there.